charts/radar-appserver/README.md

```diff
@@ -0,0 +1,66 @@
+
+
+# radar-appserver
+
+![Version: 0.0.1](https://img.shields.io/badge/Version-0.0.1-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 3.2.0](https://img.shields.io/badge/AppVersion-3.2.0-informational?style=flat-square)
+
+A Helm chart for the backend application of RADAR-base Appserver
+
+**Homepage:** <https://radar-base.org>
+
+## Maintainers
```

Comment:
Could you please review the maintainers section? I’m not working on appserver at the moment